stubs/Deprecated/@tests/stubtest_allowlist.txt

```diff
@@ -0,0 +1,3 @@
+deprecated.sphinx.SphinxAdapter.__init__
+deprecated.sphinx.versionadded
+deprecated.sphinx.versionchanged
```

Comment:
We also need to update the stub uploader not to copy the `@tests` directory. We should probably just ignore any directory starting with `@` for future compatibility.